### PR TITLE
New version: Elgato.CameraHub version 1.5.1.887 [FP-O]

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Successful
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Elgato Camera Hub |     |
| Version     | 1.5.1.887     |  |
| Publisher   | Elgato   |       |
| ProductCode |           |     |
#### Auto-updated by [vedantmgoyal2009/winget-pkgs-automation](https://github.com/vedantmgoyal2009/vedantmgoyal2009/tree/main/winget-pkgs-automation) in workflow run [4590](<https://github.com/vedantmgoyal2009/vedantmgoyal2009/actions/runs/3705297391>)

### CODE REVIEW

**Review by ItzLevvie — DISMISSED** [review]

**Comment by ItzLevvie** [comment]
@msftbot Area-External

**Comment by ItzLevvie** [comment]
@msftbot Blocking-Issue

File: manifests/e/Elgato/CameraHub/1.5.1.887/Elgato.CameraHub.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayVersion: 1.3.0.595
```
Comment (stephengillie) [inline manifests/e/Elgato/CameraHub/1.5.1.887/Elgato.CameraHub.installer.yaml:17]:
```suggestion
- DisplayVersion: 1.5.1.887
```

Comment (stephengillie) [reply manifests/e/Elgato/CameraHub/1.5.1.887/Elgato.CameraHub.installer.yaml:17]: I'm not sure why validation is erroring that the Apps & Features version is different, as it matches in my VM. 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**